### PR TITLE
Homework for lesson_02

### PR DESCRIPTION
**Pull request by TarasVolianskyi**

### CODE REVIEW

File: lesson_02/.DS_Store
Comment (tmikhailovska) [inline lesson_02/.DS_Store:1]: this file not need to store in git

File: lesson_02/convert_to_avro.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+            try:
+                convert_json_to_avro(sales_data, avro_file_path)
+            except Exception as e:
+                return jsonify({'error': f'Помилка при записі Avro-файлу: {avro_file_name}, помилка: {str(e)}'}), 500
+
+    return jsonify({'message': 'Дані успішно конвертовані у Avro'}), 200
+
+if __name__ == '__main__':
+    app.run(port=8082)
```
Comment (tmikhailovska) [inline lesson_02/convert_to_avro.py:58]: should be host value, not clear understanding that host you used

File: lesson_02/main.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    return jsonify({"message": "Data saved successfully"}), 200
+
+# Маршрут для обробки GET-запиту на кореневу сторінку
+@app.route('/', methods=['GET'])
+def home():
+    return "Welcome to the Sales API", 200
+
+# Запуск Flask-сервера
+if __name__ == '__main__':
+    app.run(port=8081)
```
Comment (tmikhailovska) [inline lesson_02/main.py:69]: the same

**Review by tmikhailovska — APPROVED** [review]
better used English language to the code comments and print

File: lesson_02/.DS_Store
Comment (TarasVolianskyi) [reply lesson_02/.DS_Store:1]: okay

**Pull request merged**